### ISSUE TITLE
In an html file with the structure: body > svg > g > foreignObject > div > div, the innermost element has an incorrect position

### ISSUE THREAD

**Issue by darthglowball** [cztomczak/cefpython#529]
I am using CEFPython with pywebview. I'm unsure if this is pywebview's fault, CEFPython or CEF's.

Test case:

```
<html>
  <head></head>
  <body>
    <svg baseProfile="full" height="1000" version="1.1" viewBox="0,0,1000,1000" width="1000" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
      <g transform="translate(0,172)" id="workspace_body">
        <foreignObject width="100%" height="100%">
          <div style="width:calc(100% - 4px);height:calc(100% - 263px);background-color:#00002d;border-style:solid;border-color:#00ffff;border-width:2" id="body_ign9375637290">
            <div style="opacity : 1; background-color : green; position : relative; width : 100; height : 100; left:0; top:0" id="div6809641435"></div>
          </div>
        </foreignObject>
      </g>
    </svg>
  </body>
</html>
```

Expected output: the green div should be in the same position as the blue div.

Issue: the (inner) green div ends up in the top-left corner of the screen. The inner div ignores its parents' positioning.

Firefox, Chrome and Opera seem to do it correctly.

My OS: Windows 7 32-bit.
CEF Version: 3.3359.1774.gd49d25f
Chromium Version: 66.0.3359.181

**Comment by cztomczak** [comment]
This is an issue with Chromium engine, but as you say it works fine in newer versions. Currently CEF Python latest release embeds Chromium v66 engine. You will have to wait for CEF Python update (no known schedule as of the moment).

**Comment by darthglowball** [comment]
@cztomczak I tried replacing some dll's in the cefpython3 folder with those found at [spotify](http://opensource.spotify.com/cefbuilds/index.html), but they either didn't fix the issue or it made CEFPython not even start. I replaced chrome_elf.dll with version 75.0.3770.80, which didn't crash my app, but it didn't fix the issue either. Is that not the chromium engine? I replaced libcef.dll with a newer version, but it crashed everything. Is it not as simple as updating your dll's?

Anyway, here is a temporary fix that works at least for the code I posted: use iframes when embedding HTML in SVG, and possibly vice versa, and possibly everytime you switch namespaces the deeper you nest.
